src/troute-network/troute/nhd_io.py

```diff
@@ -1249,6 +1249,96 @@ def get_obs_from_timeslices(
     return observation_df_new
 
 
+def get_GL_obs_from_timeslices(
```

Comment:
This function is very similar to `get_obs_from_timeslices` function. Is it possible to merge these two?

Reply:
Yeah, it's actually identical just without the interpolation step. We could merge them, but I do want to avoid interpolating for the Great Lakes, so we'd have to alter the original function to only interpolate under certain conditions. I chose this method for now for simplicity, but I'm open to re-working this function.